gdk/commands/component/WizardCommand.py

```diff
@@ -0,0 +1,16 @@
+from gdk.wizard.Prompter import Prompter
+from gdk.commands.Command import Command
+import sys
+
+
+class WizardCommand(Command):
+    def __init__(self, command_args) -> None:
+        super().__init__(command_args, "wizard")
+
+    def run(self):
+        wizard = Prompter()
+        wizard.prompt_fields()
+        wizard.utils.write_to_config_file(
+            wizard.field_dict, wizard.project_config_file
+        )
+        sys.exit("Wizard completed. Exiting...")
```

Comment:
This will raise an exception and we don't want that here. Log here that the config file is updated and exit.

Reply:
Got it